ui/app/adapters/sync/destination.js

```diff
@@ -12,8 +12,13 @@ export default class SyncDestinationAdapter extends ApplicationAdapter {
     return `${this.buildURL()}/sys`;
   }
 
-  // id is the destination name
   // modelName is sync/destinations/:type
+  urlForCreateRecord(modelName, snapshot) {
+    const { name } = snapshot.attributes();
+    return `${this._baseUrl()}/${modelName}/${name}`;
+  }
```

Comment:
@zofskeez  - The destination is returned from create but we're left with a model without an id in the store. I'm wondering if doing the following to set the id would save us from an extra request?

```js
createRecord(store, type, snapshot) {
    const { name, type: destinationType } = snapshot.attributes();
    const url = `${this._baseUrl()}/sync/destinations/${destinationType}/${name}`;

    return this.ajax(url, 'POST', { data: snapshot.serialize() }).then((resp) => ({
      id: `${destinationType}/${name}`,
      ...resp,
    }));
  }